File: programmers/난이도별/level01.두_정수_사이의_합/JiaLee0707.c
#include <stdio.h>
#include <stdbool.h>
#include <stdlib.h>

long long solution(int a, int b) {
    long long answer = 0;
    int temp;

    if (a >= b) {
        temp = a;
        a = b;
        b = temp;
    }
    for (int i = a; i <= b; i++) {
        answer += i;
    }

    return answer;
}

File: programmers/난이도별/level01.두_정수_사이의_합/JiaLee0707.java
class Solution {
    public long solution(int a, int b) {
        long answer = b;
        int z = 1;
        if(a > b) z = -1;
        for(int i=a; i!=b; i+=z) {
            answer += i;
        }
        
        return answer;
    }
}
